### ISSUE TITLE
Quantifying Justice News

### ISSUE THREAD

**Issue by tjakester** [chihacknight/breakout-groups#61]
### About the group (active tonight)
The Chicago Justice Project has had a project ongoing for five years where we are scraping Chicago media sites to capture crime, violence, and justice news. We want to capture this news, code it, quantify it, and compare to crime as it occurs in Chicago.

### Group leaders
Tracy Siska, CJP
tsiska@chicagojustice.org
3126171560

### Who we're looking for
We are looking to update our scrapers, using the 35,000 articles we have captured and coded to install natural language processing to automate the coding. 

### Tools
Unsure

### Relevant Links
http://ec2-54-88-218-235.compute-1.amazonaws.com

### Where we meet
In Gather.Town on the Giant Sea Turtle: https://gather.town/app/3JdykFv5twqLZMYQ/chn-library

**Comment by stevevance** [comment]
I want to know more about this project because I want to scrape news articles that discuss completely different topics. 

Are the scrapers hosted on GitHub?

**Comment by tjakester** [comment]
Steve,

Sorry this end up in my spam folder. Why don’t we talk about his at Hack Night next week?

Tracy



------------
Tracy Siska | Executive Director
Chicago Justice Project | 220 N. Green St., Suite # 3059 | Chicago, IL 60607
Ph. (312) 971-6745 | 

tsiska@chicagojustice.org | www.chicagojustice.org |
Twitter: CJPJustProj

**Comment by aidenkent13** [comment]
Hey @tjakester this groups sounds super cool. Have you folks gotten feedback on your projects from users yet, or people who have benefitted from your project?

**Comment by tjakester** [comment]
@aidenkent13 No. we are pretty far away from working on the front end. We still have a lot of work to do on the natural language processing and the scrapers before we get there.

**Comment by npwinkler** [comment]
Notes from tonight, if anyone wants them - https://www.evernote.com/shard/s521/sh/3e362deb-2736-40dc-962e-709882d22b15/5b82ae8d3d70ed2600dfc8b757fdd2d9

**Comment by derekeder** [comment]
Following along from the Demo night tonight, here's some helpful links to share:

* Github repo: https://github.com/chicago-justice-project/article-tagging
* Archive of scraped and tagged articles: http://data.chicagojustice.org/articles/
* Background on Justice Media Project: https://chicagojustice.org/research/justice-media-project/
* Chi Hack Night presentation (video): https://chihacknight.org/events/2018/05/01/the-justice-media-project.html
* Chi Hack Night project page: https://chihacknight.org/projects/2018/05/01/justice-media-project.html

**Comment by derekeder** [comment]
this group hasn't met in a while. closing

**Comment by derekeder** [comment]
is this group still active? if not we will close this issue until you meet again.
